File: web/src/models/DTOs/AgenciaBancoConta/AgenciaDTO.ts
import { BancoDTO } from "./BancoDTO.ts";

export interface AgenciaDTO {
  id: number;
  banco: BancoDTO;
  cnpj: string;
  numero: string;
  digitoVerificador: string;
  nome: string;
  cidade: string;
  dataInclusao: string;
  idOperadorInclusao: number;
  dataAlteracao: string;
  idOperadorAlteracao: number;
}

File: webservices/src/main/java/fieg/modulos/cadastro/agencias/dto/AgenciaDTO.java
package fieg.modulos.cadastro.agencias.dto;

import fieg.core.util.Mascaras;
import fieg.modulos.cadastro.bancos.dto.BancoDTO;
import lombok.Data;

import java.time.LocalDateTime;

@Data
public class AgenciaDTO {

    private Integer id;
    private BancoDTO banco;
    private String cnpj;
    private String numero;
    private Character digitoVerificador;
    private String nome;
    private String cidade;
    private LocalDateTime dataInclusao;
    private Integer idOperadorInclusao;
    private LocalDateTime dataAlteracao;
    private Integer idOperadorAlteracao;

    public String getCnpj() {
        return Mascaras.formataCpfCnpj(cnpj);
    }
}
